django-stubs/db/models/sql/compiler.pyi

```diff
@@ -13,9 +13,9 @@ from django.db.models.sql.subqueries import AggregateQuery, DeleteQuery, InsertQ
 from django.utils.functional import cached_property
 from typing_extensions import TypeAlias
 
-_ParamT: TypeAlias = str | int
+_ParamT: TypeAlias = Any
```

Comment:
This does not look related to EmailFiled change

Reply:
Data from another PR accidentally got here, I'm sorry)